MozillaFoundation/foundation.mozilla.org
# Update `Spotlight` homepage posts to `Ideas`, including cta link text

# Description

This PR updates the homepage `Spotlight` posts, renaming the block to `Ideas` and adding a `CTA Link Text` field, which is displayed on the lower part of the post component.

Current state:
<img width="1170" alt="image" src="https://github.com/user-attachments/assets/6e3b13f8-6a43-4e21-8755-79cb1b5b5892">
<img width="955" alt="image" src="https://github.com/user-attachments/assets/4e104b56-01f3-4ae6-ab17-b7cf7acf2397">

Updated state:
<img width="1148" alt="image" src="https://github.com/user-attachments/assets/ad5ed5c6-b615-45a9-a506-4edb4013e2b9">
<img width="933" alt="image" src="https://github.com/user-attachments/assets/d3312d84-64da-4b89-bc6c-d6c2f93782ae">

# To test

1. Log in to the [preview app CMS](https://foundation-s-tp1-981-ad-tfr9ki.herokuapp.com/cms/login/?next=/cms/) with admin2:admin2 credentials.
2. Browse the [homepage edit page](https://foundation-s-tp1-981-ad-tfr9ki.herokuapp.com/cms/pages/46/edit/). CTA Link Text be the last field input inside the `Ideas` card. Publish.
3. [Homepage](https://foundation-s-tp1-981-ad-tfr9ki.herokuapp.com/en/) should be updated with the `Ideas` header and CTA latest values.

Link to sample test page: https://foundation-s-tp1-981-ad-tfr9ki.herokuapp.com/
Related PRs/issues: #12601

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1000)
